miniapp/build.gradle

```diff
@@ -96,12 +71,23 @@ dependencies {
 
 }
 
-tasks.whenTaskAdded { task ->
-  preBuild.dependsOn 'formatKotlin'
+apply from: "../config/quality/checkstyle/android.gradle"
+apply from: '../config/quality/detekt/android.gradle'
+dependencies {
+  detektPlugins "io.gitlab.arturbosch.detekt:detekt-formatting:1.1.1"
+}
+
+apply from: '../config/quality/jacoco/android.gradle'
+jacoco {
+  toolVersion = "0.8.3"
 }
 
+apply from: '../gradle/documentation.gradle'
+
 //publishing {
 //  publications {
 //    miniapps(MavenPublication, androidArtifact(artifactId: 'miniapp-platform'))
 //  }
-//}
\ No newline at end of file
+//}
+// Android plugin - For publishing to artifactory.
+//apply from: "$CONFIG.internalConfigDir/publish/android.gradle"
```

Comment:
You can remove this one. The publishing setup can come from the normal config submodule, like this: https://github.com/rakutentech/android-sdkutils/blob/master/sdk-utils/build.gradle#L49

Reply:
The publishing part is to follow in the next PR. I didn't touch much there yet. 
I have made a note of it. Thanks.